### DISCUSSION TITLE
How to resize the random noise input of SinGAN?

### DISCUSSION THREAD

**Discussion by DonghweeYoon** [open-mmlab/mmgeneration#449] [How-to] [answered]
Hi,

Like the paper "Positional Encoding as Spatial Inductive Bias in GANs",
I would like to resize the random noise input of SinGAN to an arbitrary size.

I want to synthesize images larger than the input(G.T) image.
How can I do this?

Thank you.

**Comment by pjh4993** [comment, marked as answer]
To resize synthesize images, you need to change size of initial noise and per layer noise

You can check SinGANMSGeneratorPE's [ https://github.com/open-mmlab/mmgeneration/blob/master/mmgen/models/architectures/singan/positional_encoding.py](url)  to resize noise
